docs/source/technical_overview.rst

```diff
@@ -350,27 +350,40 @@ This approach has several advantages over an inheritance-based model:
 The Reduction Process
 ---------------------
 
-The reduction process in pyMOR is handled by so called :mod:`~pymor.reductors`
-which take arbitrary |Models| and additional data (e.g. the reduced
-basis) to create reduced |Models|. If proper offline/online
-decomposition is achieved by the reductor, the reduced |Model| will
-not store any high-dimensional data. Note that there is no inherent distinction
-between low- and high-dimensional |Models| in pyMOR. The only
+The reduction process in pyMOR is handled by so called :mod:`~pymor.reductors`,
+where each reductor takes arbitrary |Models| of associated type, and additional
+data (e.g. the reduced basis) to create reduced |Models|. For instance, the
+:class:`~pymor.reductors.basic.StationaryRBReductor` and
+:class:`~pymor.reductors.basic.InstationaryRBReductor` reductors can reduce any
+:class:`~pymor.models.basic.StationaryModel` and :class:`~pymor.models.basic.InstationaryModel`,
+while the :class:`~pymor.reductors.bt.GenericBTReductor` reduces
```

Comment:
```suggestion
while the :class:`~pymor.reductors.bt.BTReductor` reduces
```

`GenericBTReductor` is not supposed to be used directly, but only its subclasses.